downloaded-logs-20210406-203942.csv

```diff
@@ -0,0 +1,1584 @@
+2021-04-06T13:30:47.693870036Z,INFO,"PORT 3001"
+2021-04-06T13:59:59.624151379Z,INFO,"cloned document with new document_id 0xa9995b48c1f41fecac9a0df5694b46eb4351e75aba8a6337762e7d8bf2637a7b, version_id 0xa9995b48c1f41fecac9a0df5694b46eb4351e75aba8a6337762e7d8bf2637a7b and _id undefined from template 0x2a041bd9014c9ea2c79d5b9d767aa98fc2e24b2a28911b28e33a8a88072a1a0e from document_id undefined, version_id undefined and _id undefined, will insert into database"
+2021-04-06T13:59:59.629438820Z,INFO,"inserted document {"
```

Comment:
Webhook not received between these two steps, that's why there is no duplicate created.